README.md

```diff
@@ -89,7 +89,10 @@ limited by the capacity of a single physical machine, lest we sacrifice the
 property of complete auditability.  The Lightning Network [\[7\]][7] can help
 scale Bitcoin transaction volume by leaving some transactions off the ledger
 completely and is well suited for micropayments and privacy preserving payment
-rails, but may not be suitable for more generalized scaling needs.
+rails, and it is suitable for more generalized scaling needs.
+<!---
+state-channels allow for generalized smart contracts in the lightning network
+--->
```

Comment:
Ok, before we get into any further detail, what does this have to do with the Lightning Network?

https://lightning.network/

> you could use the lightning state network to send an asset linked to the price of gold from one chain to another. bitcoin and litecoin don't off state-channels, they only have payment channels.

Where is the canonical record of who owns how many of this asset, and what keeps the total amount of this asset from inflating beyond what it should?